### PR TITLE
update translations (#760)

### PR DESCRIPTION
**Pull request by tomkralidis**
Addresses #760 

@GeoSander / @pvgenuchten I couldn't figure out how to add new strings to the .po files, any guidance here?

### CODE REVIEW

**Comment by pvgenuchten** [comment]
Tom, were you successfull in retrieving the new strings? some guidance is in the pygeoapi docs

**Comment by tomkralidis** [comment]
@pvgenuchten from the docs, I tried running `pybabel update -d locale -l fr -i locale/messages.pot` but there is no such file.  Then I tried running `pybabel update -d locale -l en -i locale/en/LC_MESSAGES/messages.po` but no meaningful updates were made to the .po file.

**Comment by pvgenuchten** [comment]
Tom, the pot files are not committed, they are generated from code using: 

```
pybabel extract -F babel-mapping.ini -o locale/messages.pot ./
```

you need those pot files in order to update the .po files (which contain the translations)

**Comment by tomkralidis** [comment]
Thanks @pvgenuchten. These steps worked.  I was confused as to whether the `locale/messages.pot` file was supposed to be in version control or not (looks like not).  I have updated per your instructions.

Ready for review again @pvgenuchten @GeoSander

File: docs/source/language.rst [hunk 1]
```diff
@@ -83,7 +83,7 @@ as a list. For example, if you wish to support American English and Canadian Fre
 
 Next, you will have to provide translations for the configured languages. This involves 3 steps:
 
-1. `Add translations for configurable text values`_ in the server YAML file;
+1. `Add translations for configurable text values`_ in the pygeoap configuration;
```
Comment (GeoSander) [inline docs/source/language.rst:86]: Note there's a typo here (pygeoap instead of pygeoapi)

File: docs/source/language.rst [hunk 2]
```diff
@@ -158,11 +158,11 @@ Translator guide
 ----------------
 
 Hardcoded strings in pygeoapi templates are translated using the Babel translation system.
-Translation files are stored on the /locale folder.
+Translation files are stored on the `/locale` folder.
 Translators can follow these steps to prepare their environment for translations.
 
 
-1. Extract from latest code the keys to be translated. These keys are captured in a .pot file.
+1. Extract from latest code the keys to be translated. These keys are captured in a `.pot` file.  Note that the `.pot` file is not to be stored in version control, but as an intermidiary file used to update `/locale/*/LC_MESSAGES/messages.po` files:
```
Comment (GeoSander) [inline docs/source/language.rst:165]: Typo (intermediary)

**Review by GeoSander — APPROVED** [review]
Thanks for the changes Tom. 
+1 for merging after fixing those typos. :)

**Pull request merged**